monitoring/advanced-tracing.mdx

```diff
@@ -86,8 +85,9 @@ def ask_question(question="What is the capital of the US?") -> str:
 
 ```
 
+{/* don't know how to make it do the same thing as python code, where you put some result in the metadata, without bigger changes in how one is supposed to use baserun.trace from ts sdk */}
```

Comment:
Ok, I think we need to add a ticket to fix the TS SDK to match what Python does and in the meantime let's leave the docs as they are now.